### ISSUE TITLE
Test for WCAG success criterion 1.2.2: Captions (Prerecorded)

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#172]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html

Captions are provided for all prerecorded audio content in synchronized media, except when the media is a media alternative for text and is clearly labeled as such.
## Scope selector

video content
## Applicable

Prerecorded audio content in synchronized media
## Not applicable
- Live video content
- Video without audio
- When the media is a media alternative for text and is clearly labeled as such
## HTML Failures
- F74: Not labeling a synchronized media alternative to text as an alternative 

OR
- F75: Providing synchronized media without captions when the synchronized media presents more information than is presented on the page

OR 
- F8: Captions omitting some dialogue or important sound effects
## HTML Techniques
- G93: Providing open (always visible) captions

OR
- G87: Providing closed captions (#93)
## Relationship between failures and techniques
- F75 and F8 can only be measured with speech-to-text technology
- G93 can only be measured with image recognition technology, as it will otherwise be a 'cantTell'. This implies this success criterion will always have the outcome: pass, not applicable or cantTell and will never fail.
- F74 has three outcomes: not applicable, pass or fail. If it passes (so video is clearly labeled as text alternative), this succes criterion is not applicable. Failing is hard to detect if we don't have a technique for G93 (open captions).
- G93: using statistics. If we cant measure G93, we could make assumptions with statistics? This technique seems rarely used for captions, so we could assume this a fail. Advanced solution: make screenshots and check with simple OCR if there is text.
- This success criterion is about prerecorded content. So we should detect if it is prerecorded and not live content. We can detect this on the kind of video format is used (streaming video formats) or if a pause button or end time is available.
